internal/run-tests.yml

```diff
@@ -112,13 +87,46 @@ jobs:
     # pass through all parameters
     ${{ insert }}: ${{ parameters }}
 
-# Windows Python 3.7 build
-- template: unit-tests-with.yml
-  parameters:
-    image_name: 'windows-2019'
-    qt_wrapper: PySide2==5.14.1
-    python_version: 3.7
-    upload_coverage: false
-    job_name: "Windows Python 3.7"
-    # pass through all parameters
-    ${{ insert }}: ${{ parameters }}
+# Note: The automation library we use is not Python 3 compliant, and our ui tests currently only
+# target Windows which means we run our automation only on Windows and Python 2 for now.
+- ${{ if eq( parameters.has_unit_tests, true ) }}:
+  # Linux Python 3.7 build
+  - template: run-tests-with.yml
+    parameters:
+      image_name: 'ubuntu-18.04'
+      qt_wrapper: PySide2==5.14.1
+      python_version: 3.7
+      job_name: "Linux Python 3.7"
+      # pass through all parameters
+      ${{ insert }}: ${{ parameters }}
+
+  # macOS Python 2.7 build
+  - template: run-tests-with.yml
+    parameters:
+      image_name: 'macos-10.14'
+      qt_wrapper: PySide2==5.14.1
+      python_version: 2.7
+      job_name: "macOS Python 2.7"
+      # pass through all parameters
+      ${{ insert }}: ${{ parameters }}
+
+  # macOS Python 3.7 build
+  - template: run-tests-with.yml
+    parameters:
+      image_name: 'macos-10.14'
+      qt_wrapper: PySide2==5.14.1
+      python_version: 3.7
+      job_name: "macOS Python 3.7"
+      # pass through all parameters
+      ${{ insert }}: ${{ parameters }}
+
+  # Windows Python 3.7 build
+  - template: run-tests-with.yml
+    parameters:
+      image_name: 'windows-2019'
+      qt_wrapper: PySide2==5.14.1
```

Comment:
Probably doesn't matter but aren't we targeting 5.12?

Reply:
Yes. I pinned the version a while back to 5.14.1 due to a regression in Qt.